package.json

```diff
@@ -43,6 +43,6 @@
   "resolutions": {
     "braces": "^3.0.3",
     "micromatch": "^4.0.8",
-    "**/eslint/cross-spawn": "^7.0.5"
+    "cross-spawn": "^7.0.5"
```

Comment:
should we bump eslint version?

![image](https://github.com/user-attachments/assets/66159bb0-6611-4a14-a021-576def4ccefd)

Reply:
We may introduce

- version mismatch, OSD is using 6.8.0
- breaking change

if we want to upgrade major version of eslint, I'd leave it until OSD bumps its eslint version.